faces.py

```diff
@@ -24,11 +24,26 @@
 from authlib.flask.client import OAuth
 from werkzeug.exceptions import HTTPException
 import psycopg2
+import sys
+from dotenv import load_dotenv
 
+load_dotenv()
+
+def getEnvVar(var_name, fallback = ""):
+    value = os.getenv(var_name) or fallback
+    if not value:
+        logging.error("".join(("'", var_name, "' value not found.",
+        " Ensure a .env or .flaskenv file is present with this",
+        " environment variable set.")))
```

Comment:
Please update this to match your excellent refactoring in `update-data.py`!

Reply:
Fixed!